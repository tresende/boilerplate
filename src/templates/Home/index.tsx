import Hello from 'components/Hello'

import * as S from './styles'

const Home = () => (
  <S.Container>
    <S.Header>
      <Hello text="Hello!" />
    </S.Header>
  </S.Container>
)

export default Home
