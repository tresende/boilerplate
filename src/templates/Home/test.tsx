import { render } from 'utils/test-utils'
import Home from '.'

describe('<Home />', () => {
  it('should open modal', () => {
    const { container } = render(<Home />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
