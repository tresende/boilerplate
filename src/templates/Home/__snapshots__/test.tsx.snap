// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should open modal 1`] = `
.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-right: 2.4rem;
}

.c0 {
  margin: 0 auto;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 2.4rem 0;
}

@media (max-width:768px) {
  .c0 {
    padding-right: 15px;
    padding-left: 15px;
  }
}

@media (min-width:1200px) {
  .c0 {
    width: 1170px;
  }
}

@media (min-width:992px) {
  .c0 {
    width: 970px;
  }
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      Hello!
    </div>
  </div>
</div>
`;
